pytorch/inference/docker/1.8.0/py3/cu111/Dockerfile.gpu

```diff
@@ -0,0 +1,156 @@
+FROM nvidia/cuda:11.1.1-cudnn8-devel-ubuntu18.04
```

Comment:
Could you try using the nvidia/cuda:11.1.1-base-ubuntu18.04 image like the training image does, and installing the necessary cuda packages as a part of the container build?

The training image is about 14 GB in size right now, which is quite large as far as inference images normally are.

The training image is much smaller than it previously was because of the move from `devel` to `base`.